coderd/database/migrations/000131_merge_oidc_account.up.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS oauth_merge_state (
```

Comment:
We should just start storing all states instead of only storing the states for oauth merges, then all of the `from_login_type` and `to_login_type` can be optionally set.